src/test/kotlin/no/nb/mlt/wls/TestcontainerInitializer.kt

```diff
@@ -3,22 +3,24 @@ package no.nb.mlt.wls
 import org.springframework.boot.test.util.TestPropertyValues
 import org.springframework.context.ApplicationContextInitializer
 import org.springframework.context.ConfigurableApplicationContext
+import org.testcontainers.containers.GenericContainer
 import org.testcontainers.containers.MongoDBContainer
 import org.testcontainers.lifecycle.Startables
 import org.testcontainers.utility.DockerImageName
 
 class TestcontainerInitializer : ApplicationContextInitializer<ConfigurableApplicationContext> {
     companion object {
         val MongoContainer = MongoDBContainer(DockerImageName.parse("mongo:4.4.22"))
+        val DummySynqContainer = GenericContainer(DockerImageName.parse("harbor.nb.no/mlt/dummy-synq:main"))
```

Comment:
To avoid failures outside of NLN network during test we might just declare a variable for Docker Image name/location so you can easily change to using a locally built one instead of pulling one from NLN's harbor instance.

Reply:
We can make it a variable, but it does not seem like we can make it a property (as properties are instantiated/injected on the class, but aren't static).

Reply:
I don't think there is too much we can do here for now, as we will require the container for future PR's